web_monitoring/tests/test_html_diff_validity.py

```diff
@@ -175,3 +175,85 @@ def test_html_diff_works_on_documents_with_no_body():
 
     assert 'combined' in results
     assert isinstance(results['combined'], str)
+
+
+def test_html_diff_works_wbm_snapshots():
```

Comment:
Would you please also add at least one test case for the *non-equal* situation? For example, the exact same test as this, but with `strict_urls=None` and asserting that `results['change_count'] == 1`.